lib/resources/example_json/r4_examples_questionnaire_response.rb

```diff
@@ -27,7 +84,58 @@ module Resources
 
     R4_QUESTIONNAIRE_RESPONSE_UPDATE ||= {
       'resourceType': 'QuestionnaireResponse',
-      'id': 'SH-12508041'
+      'id': 'SH-12508041',
+      'questionnaire': 'https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Questionnaire/SH-12508041',
+      'status': 'in-progress',
+      'subject': {
+        'reference': 'Patient/12508041'
+      },
+      'item': [
+        {
+          'id': '111',
+          'linkId': '93',
+          'item': [
+            {
+              'linkId': '93-123',
+              'answer': [
+                {
+                  'valueQuantity': {
+                    'value': '20'
+                  }
+                }
+              ]
+            },
+            {
+              'linkId': '93-456',
+              'answer': [
+                {
+                  'valueString': 'text'
+                }
+              ]
+            },
+            {
+              'linkId': '93-789',
+              'extension': [
+                {
+                  'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/choice-answer',
+                  'valueBoolean': true
+                }
+              ],
+              'answer': [
+                {
+                  'valueCoding': {
+                    'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/nomenclature',
+                    'code': '960439'
+                  }
+                },
+                {
+                  'valueString': 'other text answer'
+                }
+              ]
+            }
+          ]
+        }
+      ]
```

Comment:
can you add a note extension on the item for the read and update examples?

Reply:
sure thing! I'm going back to add the extensions now

Reply:
https://github.com/cerner/fhir.cerner.com/pull/669/commits/fc013ad5d295a324502c0853abce974925c939f8